# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tinyfugue/hotfix-icu.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Use the ICU preprocessor flags only: icu-config --cflags returns -std=c11,
hiding POSIX functions like strdup and popen.

--- tinyfugue-5.2.2/unix/vars.mak.in	2025-10-27 17:50:44.000000000 +0100
+++ tinyfugue-5.2.2/unix/vars.mak.in	2026-09-27 18:19:18.444711053 +0200
@@ -32,7 +32,7 @@ SYMLINK    = @SYMLINK@
 # MANTYPE    = @MANTYPE@
 MODE       = @MODE@
 CC         = @CC@
-CFLAGS     = @CFLAGS@ @CPPFLAGS@ @LUA_DEFINES@ @LUA_INCLUDE@ @PYTHON_INCLUDE@ @ICU_CFLAGS@
+CFLAGS     = @CFLAGS@ @CPPFLAGS@ @LUA_DEFINES@ @LUA_INCLUDE@ @PYTHON_INCLUDE@ @ICU_CPPFLAGS@
 LDFLAGS    = @LDFLAGS@ 
 STD_C      = @STD_C@
 STRIP      = @STRIP@
